_posts/2018-06-05-random-vue-tips.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Random Vue Tips"
+date: 2018-06-05 10:17:26
+tags:
+- Vue
+- Development
+- JavaScript
+categories: development
+twitter_text: "Random Vue Tips"
+authors:
+- Chidozie Oragwu
+image: https://farm9.staticflickr.com/8329/8148011490_b9fb5fcf46_k_d.jpg
+image_url: https://www.flickr.com/photos/dasprid/
+image_credit: Ben Scholzen
+---
+
+[Vue](https://vuejs.org/) has been a delightful framework to work with. The first thing I noticed about it was how easy to follow the documentation was and how quickly it was to get started - just add a script reference to your page and you are ready to write code. 
+
+Being that I have only used it for less than a year, I wanted to share some things I found to be useful that will hopefully be useful to others. Here they are in no particular order:
+
+### Scenario: Arbitrarily execute component methods
```

Comment:
After running this locally I recommend `##` here (and, in the other similar places).